tests/test_integration.py

```diff
@@ -0,0 +1,45 @@
+EXPECTED_OUTPUT = """
+polygon
+├── mainnet
+│   └── geth  (default)
+├── mumbai
+│   └── geth  (default)
+└── local  (default)
+    └── test  (default)
+ethereum  (default)
+├── mainnet
+│   └── geth  (default)
+├── ropsten
+│   └── geth  (default)
+├── kovan
+│   └── geth  (default)
+├── rinkeby
+│   └── geth  (default)
+├── goerli
+│   └── geth  (default)
+└── local  (default)
+    ├── geth
+    └── test  (default)
+""".strip()
+
+
+def assert_rich_text(actual: str, expected: str):
```

Comment:
because of the issue we are seeing with running tests locally, perhaps we should adjust this.
See some of the changes in this PR! https://github.com/ApeWorX/ape/pull/920

Reply:
Standardizing the tests across fantom/polygon/bsc